lua/swenv/api.lua

```diff
@@ -23,27 +44,43 @@ end
 M.get_venvs = function(venvs_path)
   local success, Path = pcall(require, 'plenary.path')
   if not success then
-    vim.notify('Could not require plenary: '..Path, vim.log.levels.WARN)
+    vim.notify('Could not require plenary: ' .. Path, vim.log.levels.WARN)
     return
   end
   local scan_dir = require('plenary.scandir').scan_dir
 
-  local paths = scan_dir(venvs_path, {depth = 1, only_dirs =true})
   local venvs = {}
+
+  -- CONDA
+  local conda_base_path = get_conda_base(vim.fn.getenv("CONDA_EXE"))
```

Comment:
```suggestion
  local conda_base_path = get_conda_base(vim.fn.getenv("CONDA_EXE")) .. "/envs"
```
and remove the string joins below.